clinica/iotools/converters/adni_to_bids/adni_modalities/adni_fmap.py

```diff
@@ -0,0 +1,335 @@
+    """
+
+    from os import path
+
+    import pandas as pd
+
+    from clinica.iotools.converters.adni_to_bids.adni_utils import paths_to_bids
+    from clinica.utils.stream import cprint
+
+    if not subjects:
+        adni_merge_path = path.join(csv_dir, "ADNIMERGE.csv")
+        adni_merge = pd.read_csv(adni_merge_path, delimiter='","')
+        adni_merge.columns = adni_merge.columns.str.strip('"')
+        subjects = list(adni_merge.PTID.unique())
+    
+    cprint(
+        f"Calculating paths of fMRI field maps (FMAPs). Output will be stored in {conversion_dir}."
+    )
+
+    images = compute_fmap_path(source_dir, csv_dir, subjects, conversion_dir)
+    
+    cprint("Paths of field maps found. Exporting images into BIDS ...")
+
+    paths_to_bids(images, destination_dir, "fmap", mod_to_update=mod_to_update)
+    rename_fmaps(destination_dir)
+
+    cprint(msg="Field maps conversion done.", lvl="debug")
+
+def compute_fmap_path(source_dir: Path, csv_dir: Path, subjs_list: list[str], conversion_dir: Path) -> pd.DataFrame:
+    """Compute the paths to fMR images.
+
+    Args:
+        source_dir: path to the ADNI directory
+        csv_dir: path to the clinical data directory
+        subjs_list: subjects list
+        conversion_dir: path to the TSV files including the paths to original images
+
+    Returns:
+        pandas Dataframe containing the path for each fmri
+    """
+    from os import path
+
+    import pandas as pd
+
+    from clinica.iotools.converters.adni_to_bids.adni_utils import (
+        find_image_path,
+        visits_to_timepoints,
+    )
+
+    fmap_col = [
+        "Subject_ID",
+        "VISCODE",
+        "Visit",
+        "Sequence",
+        "Scan_Date",
+        "Study_ID",
+        "Field_Strength",
+        "Series_ID",
+        "Image_ID",
+    ]
+    fmap_df = pd.DataFrame(columns=fmap_col)
+    fmap_dfs_list = []
+    adni_merge = pd.read_csv(csv_dir / "ADNIMERGE.csv", sep=",", engine='python')
+
+    mayo_mri_qc = pd.read_csv(
+        path.join(csv_dir, "MAYOADIRL_MRI_IMAGEQC_12_08_15.csv"),
+        sep=",",
+        low_memory=False,
+    )
+    mayo_mri_qc = mayo_mri_qc[mayo_mri_qc.series_type == "fMRI"]
+    mayo_mri_qc.columns = [x.upper() for x in mayo_mri_qc.columns]
+
+    mayo_mri_qc3 = pd.read_csv(
+        path.join(csv_dir, "MAYOADIRL_MRI_QUALITY_ADNI3.csv"), sep=",", low_memory=False
+    )
+    mayo_mri_qc3 = mayo_mri_qc3[mayo_mri_qc3.SERIES_TYPE == "EPB"]
+
+    # Concatenating visits in both QC files
+    mayo_mri_qc = pd.concat(
+        [mayo_mri_qc, mayo_mri_qc3], axis=0, ignore_index=True, sort=False
+    )
+
+    mri_list = pd.read_csv(path.join(csv_dir, "MRILIST.csv"), sep=",", low_memory=False)
+
+    # Selecting only fMRI images that are not Multiband
+    mri_list = mri_list[
+        mri_list.SEQUENCE.str.contains("apping")
+    ]  # 'apping' includes all field map scans, but not others
+
+    for subj in subjs_list:
+        ssubj = subj.replace('"','')
```

Comment:
Would this be necessary if you were relying on `load_clinical_csv()` for instance ?